test/unit/controllers/category-controller-spec.js

```diff
@@ -40,7 +40,7 @@
       });
 
       it('providers should exist', function () {
-        expect(ctrl.providers).to.exist;
+        expect(ctrl.providers).to.exist; //jshint ignore:line
```

Comment:
el error de jshint Expected an assignment or function call and instead saw an expression.